book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -444,43 +396,37 @@ Finally, we load the file containing the position of each sprite on the sheet.
 
 ```rust
 # use amethyst::{
-#   assets::{AssetStorage, DefaultLoader, Handle, Loader},
-#   core::transform::Transform,
-#   prelude::*,
-#   renderer::{
-#       camera::Camera,
-#       formats::texture::ImageFormat,
-#       sprite::{SpriteRender, SpriteSheet, SpriteSheetFormat},
-#       Texture,
-#   },
+#     assets::{DefaultLoader, Handle, Loader, ProcessingQueue},
+#     core::transform::Transform,
+#     prelude::*,
+#     renderer::{Camera, SpriteRender, SpriteSheet, Texture},
 # };
 # 
-fn load_sprite_sheet(world: &mut World) -> Handle<SpriteSheet> {
-#   let texture_handle = {
-#       let loader = resources.get::<DefaultLoader>();
-#       let texture_storage = resources.get::<AssetStorage<Texture>>();
-#       loader.load(
-#           "texture/pong_spritesheet.png",
-#       )
+fn load_sprite_sheet(resources: &mut Resources) -> Handle<SpriteSheet> {
+#   // Load the sprite sheet necessary to render the graphics.
+#   // `texture` is a cloneable reference to the pixel data.
+#   let texture: Handle<Texture> = {
+#       let loader = resources.get::<DefaultLoader>().unwrap();
+#       loader.load("texture/pong_spritesheet.png")
 #   };
 # 
-    // ...
+    // --snip--
 
     let loader = resources.get::<DefaultLoader>();
-    let sprite_sheet_store = resources.get::<AssetStorage<SpriteSheet>>();
-    loader.load(
-        "texture/pong_spritesheet.ron", // Here we load the associated ron file
-        SpriteSheetFormat(texture_handle),
+    let sprites = loader.load("texture/pong_spritesheet.ron");
+    loader.load_from_data(
```

Comment:
I don't understand what the second and third args to `load_from_data` are used for. It might be nice to explain them here.